### ISSUE TITLE
[Feature request name]:

### ISSUE THREAD

**Issue by barron-charles** [department-of-veterans-affairs/va.gov-team#67576]
### Describe the problem

Veterans don't have a way to say they are homeless in their VA profile. When a homeless Veteran is filling out their form on VA.gov, there isn't a pattern or specific recommendation in the VA design system to indicate they are homeless. 

When the Veteran is homeless, the address section might not be applicable to the Veteran applying. Providing a clear solution to this problem could enhance Veteran user experience, providing a more empathetic approach for the Veteran's special cares and needs. 

### Who will benefit

Veterans, service members, or family members who are homeless trying to fill out a form on VA.gov.

### Describe your idea

The pattern might involve a checkbox for the person's contact information, indicating the person filling out the form is homeless. It could also include a separate screen, having questions giving the user the ability to indicate they don't have previous address information at the moment. 

Some additional context– a homeless person could submit an application for PO Box™ service to a local Post Office, or use a General Delivery as a temporary mailing address, meaning they would still have a temporary mailing address to input information into.

The screenshot used is might be an idea to implement that is taken from the VA disability application (https://www.va.gov/disability/file-disability-claim-form-21-526ez/introduction).

<img width="441" alt="Screenshot 2023-10-13 at 12 04 22 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95768999/b8c4fa77-c9e6-4a48-805a-9f5744cc191e">

### Provide evidence

According to VA.gov (https://www.va.gov/homeless/pit_count.asp),"The national snapshot of Veteran homelessness showed that the total number of Veterans who experienced homelessness was 33,129 – a decrease of 11% over January 2020, the last year a full PIT Count was conducted."

This small change could be a large step in addressing Veteran's needs.

### Platform Mission

- [X] Keeps these products and services available, stable, and secure.
- [X] Ensures they are consistent and meet high standards of quality in Veteran-centered user experience and technical approach.
- [ ] Provides shared tools, software, support, and technical infrastructure to accelerate their development.
- [ ] Other

**Comment by humancompanion-usds** [comment]
The need for this pattern was raised a little over a year ago by ShawnaH before she departed for Code for America. Unfortunately, no issue was filed and the idea has sat on a Mural board for awhile. Thus it's been hanging out in our pattern queue but with no forward momentum. So thank you for filing this issue.

I'm all for creating a pattern for this. I actually worked on Emergency Rental Assistance during the pandemic for a year and designed a fform to collect housing instability information](https://home.treasury.gov/policy-issues/coronavirus/assistance-for-state-local-and-tribal-governments/emergency-rental-assistance-program/service-design/example-application-screens#risk-of-housing-instability). I mention this because I think we could adapt the language that HUD recommends and that Treasury approved for that project. It's a bit more complicated than "Are you homeless? yes/no".

What we've lacked is a team to test it out. This is definitely going to require research with Veterans to get their reaction to the language and options. If no VFS team steps forward I can eventually hand this to the Veteran-facing Forms team to test, but their queue is full for this quarter and the next. Thus if you are a team that can take this on or know of one, that would help.

**Comment by barron-charles** [comment]
Yeah, no problem. I see what you meant with having a bit of additional context for their potentially trauma-inducing homeless circumstance. Did a great job with the Emergency Rental Assistance form avoiding asking tenants to provide a narrative. That extra background could be considered uncomfortable for users to answer, but is important for understanding the entire user situation.

We might include similar checkboxes to find information, or just provide content for what it means to be homeless. Will talk with @ccruzgra about the ability for us to pick this one up this quarter further too.

**Comment by ccruzgra** [comment]
@humancompanion-usds We were advised that, since they've never received a pre-need request from a homeless applicant, this wouldn't be something our team could pick up as work to test out at this time.

**Comment by caw310** [comment]
@tayism Matt might be working on this. It isn't currently planned in a DST sprint.

**Comment by gopixelsgo** [comment]
Hi @humancompanion-usds - we're seeing this item is **In Progress - Has Owner** in the Design Gaps board. Are there any new updates we'd want to reflect here? Thanks!

**Comment by humancompanion-usds** [comment]
I sure do own it. I have what I need to write the pattern except for time.

**Comment by gopixelsgo** [comment]
Thank you @humancompanion-usds !

**Comment by gopixelsgo** [comment]
Hi @humancompanion-usds - any update on this item? Thanks!

**Comment by humancompanion-usds** [comment]
This is the issue where @jeana-adhoc had all of the good details that I need to write this. Found it and now I'm sitting down to write this guidance.

**Comment by humancompanion-usds** [comment]
[PR is up for review](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/3043).

**Comment by humancompanion-usds** [comment]
Pattern is deployed!: https://design.va.gov/patterns/ask-users-for/housing-status

> there isn't a pattern or specific recommendation in the VA design system to indicate they are homeless

There is now a pattern. However, how directly applicable that pattern is to VA.gov Profile is a different matter. Profile has to deal with back-end systems that may not have a place to store this information. I must admit I'm ignorant of how this might be managed in terms of both capturing that a Veteran is experiencing housing instability and allowing for the lack of a mailing address. Tagging @travis-newby who may have some insight. Travis - Happy to have a discussion on this topic to figure out how we might scale this work.

**Comment by jamiefiore** [comment]
@humancompanion-usds  @travis-newby  I was poking through the feature requests stumbled across this one. This seemed like a good place to share what the IVC came up with earlier on in the CHAMPVA program. Hopefully it's relevant or helpful. My team worked on some solutioning for how a person can complete a mailing address form when they don't have stable housing. Some programs, like CHAMPVA and FMP require mail to be sent our their beneficiaries so the user has to fill in someplace for the mail to go. Most people will use a friend or relative's address. Some can use either a homeless shelter or a VA office but we found the most reliable address for most types of situations was to use General Delivery at a local post office.  

After we did this desk research we developed some content for an additional component on the already existing address pattern to solve this problem. 
![Image](https://github.com/user-attachments/assets/b3ff26e6-e444-44a4-ace7-9e011e1d3b14)
The General Delivery link would go to the [USPS webpage](https://faq.usps.com/s/article/What-is-General-Delivery) that explains how to use and add General Delivery. 

Hope this helps or at least generates some ideas.

**Comment by humancompanion-usds** [comment]
@danbrady and @caw310 - Can we get the above comment from Jamie added to the pattern? Thanks!

**Comment by caw310** [comment]
We have created a new ticket to add comment from Jamie to the pattern. [Update Address Pattern for Users Without Mailing Addresses](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3496)

**Comment by jamiefiore** [comment]
I just saw this now. Thank you @humancompanion-usds @caw310 @danbrady. Let me know if you need anymore info about this.

**Comment by tbaker1026** [comment]
@humancompanion-usds does this [guidance](https://design.va.gov/patterns/ask-users-for/housing-status) meet the acceptance criteria to close this one out or do we need further investigation on how to handle mailing address in Profile?

**Comment by humancompanion-usds** [comment]
@tbaker1026 - No, the Ask users for housing status pattern doesn't cover the VA Profile on VA.gov. That's what this issue is asking for. We could transfer this issue to Chante's team. Profile is their domain.
